libs/domain/cart/entries/src/entries.component.ts

```diff
@@ -1,169 +1,67 @@
-import { CartComponentMixin, CartEntry, CartService } from '@spryker-oryx/cart';
-import { resolve } from '@spryker-oryx/di';
-import { ContentController } from '@spryker-oryx/experience';
-import { ProductService } from '@spryker-oryx/product';
-import { asyncValue, hydratable } from '@spryker-oryx/utilities';
-import { html, TemplateResult } from 'lit';
+import { CartComponentMixin } from '@spryker-oryx/cart';
+import { ContentMixin, defaultOptions } from '@spryker-oryx/experience';
+import { hydratable, i18n } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
 import { repeat } from 'lit/directives/repeat.js';
-import { BehaviorSubject, combineLatest, map, of, switchMap, tap } from 'rxjs';
+import { RemoveByQuantity } from '../../entry/src';
 import { CartEntriesOptions } from './entries.model';
-import { styles } from './entries.styles';
-
-interface EntriesData {
-  entries: CartEntry[];
-  loading: boolean;
-  currentlyUpdated: string | null;
-  options: Partial<CartEntriesOptions>;
-}
-
+import { cartEntriesStyles } from './entries.styles';
+
+@defaultOptions({
+  removeByQuantity: RemoveByQuantity.ShowBin,
+  enableItemId: true,
+  enableItemImage: true,
+  enableItemPrice: true,
+} as CartEntriesOptions)
```

Comment:
We don't need typecasting here

Reply:
but very useful as long as we refactor; keeping it until we have proper typing in place.